src/components/Navigation/Ui/ButtonWithTooltip.js

```diff
@@ -3,9 +3,9 @@
 import React from 'react'
 import { Tooltip } from 'antd'
 
-const ButtonWithTooltip = ({ button, tip }) => {
+const ButtonWithTooltip = ({ button, tip, disabled }) => {
   return (
-    <Tooltip placement="topLeft" title={tip} arrowPointAtCenter>
+    <Tooltip placement="topLeft" title={!disabled && tip} arrowPointAtCenter>
```

Comment:
I think this is a nice workaround for the issue!

Can we read the disabled property of `button` or do we need a separate prop for this?

Reply:
don't think that we can use the disabled prop of the supplied button. In the general case the value fo the `button` prop may not be an instance of the `Button` component so may not have a `disabled` prop. For example, it might be a Button wrapped in some other component (e.g., a Dropzone). So `button` would be an instance of Dropzone.